From b7e91abfd4b4535c0021191c9abe81040d27451c Mon Sep 17 00:00:00 2001
From: Tommy Webb <tommy@calyxinstitute.org>
Date: Wed, 8 Mar 2023 15:19:00 -0500
Subject: [PATCH] Remove "Managed browser" from menu temporarily

This is a temporary change to prevent users from being confused or
learning to ignore the incorrect "Managed browser" menu item.

TODO: Properly implement a way to differentiate between our policies
and policies set via some other mechanism, e.g. actual Device Policy.

Change-Id: Iabdf615be5fb0f0d65fb66d8de0cb8640aa57bce
---
 .../browser/tabbed_mode/TabbedAppMenuPropertiesDelegate.java     | 1 +
 1 file changed, 1 insertion(+)

diff --git a/chrome/android/java/src/org/chromium/chrome/browser/tabbed_mode/TabbedAppMenuPropertiesDelegate.java b/chrome/android/java/src/org/chromium/chrome/browser/tabbed_mode/TabbedAppMenuPropertiesDelegate.java
index ac55533fcbb7e..4fe20b046e55e 100644
--- a/chrome/android/java/src/org/chromium/chrome/browser/tabbed_mode/TabbedAppMenuPropertiesDelegate.java
+++ b/chrome/android/java/src/org/chromium/chrome/browser/tabbed_mode/TabbedAppMenuPropertiesDelegate.java
@@ -116,6 +116,7 @@ public class TabbedAppMenuPropertiesDelegate extends AppMenuPropertiesDelegateIm
 
     @Override
     protected boolean shouldShowManagedByMenuItem(Tab currentTab) {
+        if (true) return false;
         Profile profile = Profile.fromWebContents(currentTab.getWebContents());
         return profile != null && ManagedBrowserUtils.isBrowserManaged(profile);
     }
-- 
2.39.2

